recipe/0101-Build-fix-in-a-conda-environment.patch

```diff
@@ -20,8 +20,8 @@ index 7ba06ff..fabef59 100755
    echo 'Using installed tensorflow.'
  else
    echo 'Installing tensorflow.'
--  pip install tensorflow==2.3.0
-+  conda install tensorflow=2.3.*
+-  pip install tensorflow==2.3.1
```

Comment:
Upstream tf text is still depending on TF 2.3.1. I suspect they will update this to 2.4.0 once TF 2.4.0 GA is tagged. Even if they do not, I think build wise it worked. We will need to test with TF 2.4.0.